src/server/master_server/master_chunk_server_manager_service_impl.h

```diff
@@ -28,6 +29,19 @@ class MasterChunkServerManagerServiceImpl final
       grpc::ServerContext* context,
       const protos::grpc::ReportChunkServerRequest* request,
       protos::grpc::ReportChunkServerReply* reply) override;
+ public:
+  // Default constructor, only used in test
+  MasterChunkServerManagerServiceImpl() : config_manager_(nullptr), 
```

Comment:
honestly config manager and resolve_hostname isn't needed here. Please see my previous comment about this.

Reply:
Ditto.